contributing.rst

```diff
@@ -1,3 +1,15 @@
+How to install for developing
+-----------------------------
+
+Clone the repository with::
+
+    git clone git@github.com:PLOS/allofplos.git
+
+Install from the downloaded directory with::
+
+    pip install -U -e /local/path/allofplos
```

Comment:
People will copy and paste commands and this will result in an error, better would be to include the `cd` command and then install using a relative path (ideally `pip install -U -e .`).